var/spack/repos/builtin/packages/netcdf-fortran/package.py

```diff
@@ -0,0 +1,18 @@
+from spack import *
+
+class NetcdfFortran(Package):
+    """Fortran interface for NetCDF4"""
+
+    homepage = "http://www.unidata.ucar.edu/downloads/netcdf/netcdf-cxx/index.jsp"
+    url      = "http://www.unidata.ucar.edu/downloads/netcdf/ftp/netcdf-fortran-4.4.3.tar.gz"
+
+    version('4.4.3', 'bfd4ae23a34635b273d3eb0d91cbde9e')
+
+    variant('mpi', default=True, description='Enables MPI parallelism')
```

Comment:
This variant appears to be unused. It could be used as follows:
```python
depends_on('netcdf', when='~mpi')
depends_on('netcdf+mpi', when='+mpi')
```
This is similar to the way that gcc works with its binutils dependency. However, I'm not sure if this is even necessary. It feels strange to make a variant that doesn't affect the way a package is built. NetCDF-Fortran already depends on MPI when it is built with `netcdf+mpi`, so Spack shouldn't have any trouble installing everything at the same time if you specify:
```bash
spack install netcdf-fortran ^netcdf+mpi ^openmpi
```
@tgamblin: Thoughts? We might just want to remove this variant altogether.